bin/ofs/Cargo.toml

```diff
@@ -39,7 +39,7 @@ futures-util = "0.3.30"
 libc = "0.2.151"
 log = "0.4.20"
 nix = { version = "0.27.1", features = ["user"] }
-opendal.workspace = true
+opendal = {path="../../core"}
```

Comment:
Ditto https://github.com/apache/opendal/pull/4134/files#r1477020121

Reply:
The same with https://github.com/apache/opendal/pull/4134#discussion_r1477048529